### PR TITLE
docs: update README for Scientific Computing with Python Certification

### LINKED ISSUE #55203: Update README for Scientific Computing with Python Certification
The README is not updated for the new version of the Scientific Computing with Python Certification.

### PR DESCRIPTION
**Pull request by ganesh-159th**
...Simplify project section

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
  - Confirm that you have reviewed and adhered to the guidelines set forth by freeCodeCamp for contributing to their projects.
  
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
  - Ensure that you have understood and followed the steps outlined in the guide for opening a pull request.
  
- [x] My pull request targets the `main` branch of freeCodeCamp.
  - Verify that your pull request is intended to merge into the `main` branch of the freeCodeCamp repository.
  
- [x] I have tested these changes either locally on my machine, or GitPod.
  - Confirm that you have tested the modifications locally or in GitPod to ensure they function correctly.

Closes #55203

Streamlined the project section for the Scientific Computing with Python Certification by focusing on core project links. This adjustment enhances readability and directs users to explore detailed project completion sections directly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,12 +131,22 @@ Here are our twelve core certifications:
   <br />
   **Projects**: [Metric-Imperial Converter](https://www.freecodecamp.org/learn/quality-assurance/quality-assurance-projects/metric-imperial-converter), [Issue Tracker](https://www.freecodecamp.org/learn/quality-assurance/quality-assurance-projects/issue-tracker), [Personal Library](https://www.freecodecamp.org/learn/quality-assurance/quality-assurance-projects/personal-library), [Sudoku Solver](https://www.freecodecamp.org/learn/quality-assurance/quality-assurance-projects/sudoku-solver), [American British Translator](https://www.freecodecamp.org/learn/quality-assurance/quality-assurance-projects/american-british-translator)
 
-#### 8. [Scientific Computing with Python Certification](https://www.freecodecamp.org/learn/scientific-computing-with-python/)
+### 8. [Scientific Computing with Python Certification](https://www.freecodecamp.org/learn/scientific-computing-with-python/)
```
Comment (ilenia-magoni) [inline README.md:134]: don't change this

File: README.md [hunk 2]
```diff
@@ -131,12 +131,22 @@ Here are our twelve core certifications:
...
 
 - [Introduction to Python for Everybody](https://www.freecodecamp.org/learn/scientific-computing-with-python/#python-for-everybody)
```
Comment (ilenia-magoni) [inline README.md:136]: you should replace this with the links to the various practice projects

File: README.md [hunk 3]
```diff
@@ -131,12 +131,22 @@ Here are our twelve core certifications:
...
   <br />
   <br />
-  **Projects**: [Arithmetic Formatter](https://www.freecodecamp.org/learn/scientific-computing-with-python/scientific-computing-with-python-projects/arithmetic-formatter), [Time Calculator](https://www.freecodecamp.org/learn/scientific-computing-with-python/scientific-computing-with-python-projects/time-calculator), [Budget App](https://www.freecodecamp.org/learn/scientific-computing-with-python/scientific-computing-with-python-projects/budget-app), [Polygon Area Calculator](https://www.freecodecamp.org/learn/scientific-computing-with-python/scientific-computing-with-python-projects/polygon-area-calculator), [Probability Calculator](https://www.freecodecamp.org/learn/scientific-computing-with-python/scientific-computing-with-python-projects/probability-calculator)
```
Comment (ilenia-magoni) [inline README.md:139]: do not change the formatting, change only the links. Also please use the links to the project page directly.

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
Thank you for trying to contribute, but please do not change the formatting, and add the links of the step based projects that have replaced Python for Everybody

File: README.md [hunk 4]
```diff
@@ -136,7 +136,7 @@ Here are our twelve core certifications:
...
+  **Projects**:[Arithmetic Formatter](https://www.freecodecamp.org/learn/scientific-computing-with-python/build-an-arithmetic-formatter-project/build-an-arithmetic-formatter-project),[Time Calculator](https://www.freecodecamp.org/learn/scientific-computing-with-python/build-a-time-calculator-project/build-a-time-calculator-project),[Budget App](https://www.freecodecamp.org/learn/scientific-computing-with-python/build-a-budget-app-project/build-a-budget-app-project),[Polygon Area Calculator](https://www.freecodecamp.org/learn/scientific-computing-with-python/build-a-polygon-area-calculator-project/build-a-polygon-area-calculator-project),[Probability Calculator](https://www.freecodecamp.org/learn/scientific-computing-with-python/build-a-probability-calculator-project/build-a-probability-calculator-project)
```
Comment (ilenia-magoni) [inline README.md:139]: can you re add the spaces as they were?

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
you still need to follow the bot suggestions, and add the practice projects links

File: README.md (hunk 2 above)
Comment (ganesh-159th) [reply README.md]: i have replace "- [Introduction to Python for Everybody](https://www.freecodecamp.org/learn/scientific-computing-with-python/#python-for-everybody)" one like with the various practice projects link please check this one in the 4 commit  one

**Review by ganesh-159th — COMMENTED** [review]
Updated links to include various practice projects for Scientific Computing with Python

File: README.md [hunk 5]
```diff
@@ -133,10 +133,12 @@ Here are our twelve core certifications:
...
 
-- [Introduction to Python for Everybody](https://www.freecodecamp.org/learn/scientific-computing-with-python/#python-for-everybody)
-  <br />
-  <br />
-  **Projects**: [Arithmetic Formatter](https://www.freecodecamp.org/learn/scientific-computing-with-python/scientific-computing-with-python-projects/arithmetic-formatter), [Time Calculator](https://www.freecodecamp.org/learn/scientific-computing-with-python/scientific-computing-with-python-projects/time-calculator), [Budget App](https://www.freecodecamp.org/learn/scientific-computing-with-python/scientific-computing-with-python-projects/budget-app), [Polygon Area Calculator](https://www.freecodecamp.org/learn/scientific-computing-with-python/scientific-computing-with-python-projects/polygon-area-calculator), [Probability Calculator](https://www.freecodecamp.org/learn/scientific-computing-with-python/scientific-computing-with-python-projects/probability-calculator)
+**Practice Projects for Scientific Computing with Python**:[Project Euler](https://projecteuler.net/archives), [Python Challenge](http://www.pythonchallenge.com/), [LeetCode](https://leetcode.com/problemset/all/), [Codewars](https://www.codewars.com/?language=python)
```
Comment (ilenia-magoni) [inline README.md:136]: What are these links? Please take it seriously.

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
Please take this seriously. Add the links to the proper practice projects of the Scientific Computing with Python certification.

File: README.md (hunk 5 above)
Comment (ganesh-159th) [reply README.md:136]: Should I replace the previous links with these updated ones for the freeCodeCamp projects?

[Arithmetic Formatter Project](https://www.freecodecamp.org/learn/scientific-computing-with-python/build-an-arithmetic-formatter-project/build-an-arithmetic-formatter-project)
[Time Calculator Project](https://www.freecodecamp.org/learn/scientific-computing-with-python/build-a-time-calculator-project/build-a-time-calculator-project)
[Budget App Project](https://www.freecodecamp.org/learn/scientific-computing-with-python/build-a-budget-app-project/build-a-budget-app-project)
[Polygon Area Calculator Project](https://www.freecodecamp.org/learn/scientific-computing-with-python/build-a-polygon-area-calculator-project/build-a-polygon-area-calculator-project)
[Probability Calculator Project](https://www.freecodecamp.org/learn/scientific-computing-with-python/build-a-probability-calculator-project/build-a-probability-calculator-project)
Please confirm.

Comment (Dario-DC) [inline README.md:136]: You should delete that and list the names of the practice projects with the respective link. For example:
`Learn String Manipulation by Building a Cipher` should link to https://www.freecodecamp.org/learn/scientific-computing-with-python/#learn-string-manipulation-by-building-a-cipher

Comment (Dario-DC) [reply README.md:136]: You should delete the lines you added above and list all the practice project with the respective link. For example:
`Learn String Manipulation by Building a Cipher` should link to https://www.freecodecamp.org/learn/scientific-computing-with-python/#learn-string-manipulation-by-building-a-cipher

Comment (ganesh-159th) [reply README.md:136]: as you told me i have done that please  check  this out "[Learn String Manipulation by Building a Cipher](https://www.freecodecamp.org/learn/scientific-computing-with-python/#learn-string-manipulation-by-building-a-cipher), [Learn How to Work with Numbers and Strings by Implementing the Luhn Algorithm](https://www.freecodecamp.org/learn/scientific-computing-with-python/#learn-how-to-work-with-numbers-and-strings-by-implementing-the-luhn-algorithm), [Learn Lambda Functions by Building an Expense Tracker](https://www.freecodecamp.org/learn/scientific-computing-with-python/#learn-lambda-functions-by-building-an-expense-tracker), [Learn List Comprehension by Building a Case Converter Program](https://www.freecodecamp.org/learn/scientific-computing-with-python/#learn-list-comprehension-by-building-a-case-converter-program), [Learn the Bisection Method by Finding the Square Root of a Number](https://www.freecodecamp.org/learn/scientific-computing-with-python/#learn-the-bisection-method-by-finding-the-square-root-of-a-number), [Learn Regular Expressions by Building a Password Generator](https://www.freecodecamp.org/learn/scientific-computing-with-python/#learn-regular-expressions-by-building-a-password-generator), [Learn Algorithm Design by Building a Shortest Path Algorithm](https://www.freecodecamp.org/learn/scientific-computing-with-python/#learn-algorithm-design-by-building-a-shortest-path-algorithm), [Learn Recursion by Solving the Tower of Hanoi Puzzle](https://www.freecodecamp.org/learn/scientific-computing-with-python/#learn-recursion-by-solving-the-tower-of-hanoi-puzzle), [Learn Data Structures by Building the Merge Sort Algorithm](https://www.freecodecamp.org/learn/scientific-computing-with-python/#learn-data-structures-by-building-the-merge-sort-algorithm), [Learn Classes and Objects by Building a Sudoku Solver](https://www.freecodecamp.org/learn/scientific-computing-with-python/#learn-classes-and-objects-by-building-a-sudoku-solver), [Learn Tree Traversal by Building a Binary Search Tree](https://www.freecodecamp.org/learn/scientific-computing-with-python/#learn-tree-traversal-by-building-a-binary-search-tree), [Learn Special Methods by Building a Vector Space](https://www.freecodecamp.org/learn/scientific-computing-with-python/#learn-special-methods-by-building-a-vector-space)
"

Comment (ganesh-159th) [reply README.md:136]: do i proceed  with this one ?

Comment (Dario-DC) [reply README.md:136]: Go ahead, please. But add them as items of an unordered list.

File: README.md [hunk 6]
```diff
@@ -133,10 +133,25 @@ Here are our twelve core certifications:
...
+- [Learn Algorithm Design by Building a Shortest Path Algorithm](https://www.freecodecamp.org/learn/scientific-computing-with-python/#learn-algorithm-design-by-building-a-shortest-path-algorithm)
+- [Learn Recursion by Solving the Tower of Hanoi Puzzle](https://www.freecodecamp.org/learn/scientific-computing-with-python/#learn-recursion-by-solving-the-tower-of-hanoi-puzzle)
+- [Learn Data Structures by Building the Merge Sort Algorithm](https://www.freecodecamp.org/learn/scientific-computing-with-python/#learn-data-structures-by-building-the-merge-sort-algorithm)
+- [Learn Classes and Objects by Building a Sudoku Solver](https://www.freecodecamp.org/learn/scientific-computing-with-python/#learn-classes-and-objects-by-building-a-sudoku-solver)
+- [Learn Tree Traversal by Building a Binary Search Tree](https://www.freecodecamp.org/learn/scientific-computing-with-python/#learn-tree-traversal-by-building-a-binary-search-tree)
+- [Learn Special Methods by Building a Vector Space](https://www.freecodecamp.org/learn/scientific-computing-with-python/#learn-special-methods-by-building-a-vector-space)
+
```
Comment (Dario-DC) [inline README.md:150]:
```suggestion
```

**Review by Dario-DC — CHANGES_REQUESTED** [review]

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
The content of the PR is much better, thank you. Now you need to follow camperbot message.

File: README.md [hunk 7]
```diff
@@ -133,10 +133,25 @@ Here are our twelve core certifications:
 
 #### 8. [Scientific Computing with Python Certification](https://www.freecodecamp.org/learn/scientific-computing-with-python/)
 
-- [Introduction to Python for Everybody](https://www.freecodecamp.org/learn/scientific-computing-with-python/#python-for-everybody)
-  <br />
-  <br />
-  **Projects**: [Arithmetic Formatter](https://www.freecodecamp.org/learn/scientific-computing-with-python/scientific-computing-with-python-projects/arithmetic-formatter), [Time Calculator](https://www.freecodecamp.org/learn/scientific-computing-with-python/scientific-computing-with-python-projects/time-calculator), [Budget App](https://www.freecodecamp.org/learn/scientific-computing-with-python/scientific-computing-with-python-projects/budget-app), [Polygon Area Calculator](https://www.freecodecamp.org/learn/scientific-computing-with-python/scientific-computing-with-python-projects/polygon-area-calculator), [Probability Calculator](https://www.freecodecamp.org/learn/scientific-computing-with-python/scientific-computing-with-python-projects/probability-calculator)
+**Practice Projects for Scientific Computing with Python**:-
+
```
[the remark this answers is not in the document]
Comment (Dario-DC) [reply README.md:137]: You should implement the suggested changes

**Review by huyenltnguyen — COMMENTED** [review]
Please kindly read https://github.com/freeCodeCamp/freeCodeCamp/pull/55204#issuecomment-2172944173 and follow the pointers there.

The title of this PR needs to be updated to follow our convention.

**Review by huyenltnguyen — APPROVED** [review]

**Review by Dario-DC — APPROVED** [review]
LGTM :+1: 

I have edited the line that enables to automatically close the issue which shouldn't have that extra #.
image

**Review by majestic-owl448 — APPROVED** [review]

**Pull request merged**